Update author page and GSoC project description (#6364)
jenkins-infra/jenkins.io: content/_data/authors/harsh-ps-2003.adoc

---
name: "Harsh Pratap Singh"
github: harsh-ps-2003
twitter: harsh_ps2003
linkedin: harsh-pratap-singh-787485255
---

> Harsh is currently a Freshman in Indian Institute of Technology, Kanpur whose interests lies in rapidly evolving Computer Science fields like DevSecOps, Blockchain and more. He is an avid Open Source Contributor and is inspired by the idea of developing useful Open Source Software for the masses to use. 

Reviewer: ```suggestion
Harsh is currently a Freshman at the Indian Institute of Technology, Kanpur, whose interests lies in rapidly evolving Computer Science fields like DevSecOps, Blockchain and more. He is an avid open-source contributor and is inspired by the idea of developing useful open-source software for the masses to use. 
```

> Other than Software development, his interest lies in Economics, Philosophy and Psychology.

Reviewer: ```suggestion
Other than software development, his interest lies in economics, philosophy, and psychology.
```

> He is a Jenkins Google Summer of Code (GSoC) contributor in 2023, associated with the link:https://github.com/jenkinsci/gitlab-plugin[GitLab Plugin Modernization] project.

Reviewer: ```suggestion
He is a Jenkins Google Summer of Code (GSoC) contributor in 2023, participating in the link:https://github.com/jenkinsci/gitlab-plugin[GitLab Plugin Modernization] project.
```

> He started his journey of contributing to Jenkins in February 2023 and got hooked.

Reviewer: ```suggestion
He started his journey of contributing to Jenkins in February 2023 and got hooked since.
```
